_graphql/config.yml

```diff
@@ -0,0 +1,9 @@
+resolvers:
```

Comment:
I get that this yaml might be used differently than all other `Config`-related yaml files, it just feels weird it doesn't have a root node/value either as an existing class or at least some string, like GraphQL or so. I know it might be redundant in terms of verbosity, but... Just a thought.

Reply:
Yeah good point. This is a matter of preference.You can also do an arbitrary file name, like `schema.yml` containing any keys you like

```yaml
config: ...
types:... 

# etc..
```